README.markdown

```diff
@@ -81,16 +81,18 @@ which should output 'nvm' if the installation was successful. Please note that `
 
 For manual install create a folder somewhere in your filesystem with the `nvm.sh` file inside it. I put mine in `~/.nvm`.
 
-Or if you have `git` installed, then just clone it, and check out the latest version:
+Or if you have `git` installed:
 
-```sh
-git clone https://github.com/creationix/nvm.git ~/.nvm && cd ~/.nvm && git checkout `git describe --abbrev=0 --tags`
-```
-
-To activate nvm, you need to source it from your shell:
+1. clone this repo
+2. check out the latest version
+3. activate nvm by sourcing it from your shell
 
 ```sh
-. ~/.nvm/nvm.sh
+(
+  git clone https://github.com/creationix/nvm.git ~/.nvm
+  cd ~/.nvm
+  git checkout `git describe --abbrev=0 --tags --match "v[0-9]*" origin`
```

Comment:
I found it in the oldest version that the git documentation tells me about, 1.8.1.6. https://git-scm.com/docs/git-describe/1.8.1.6

I wanted an answer, so I cloned git and used git blame and traced it back to 30ffa603 from 2007. https://git.kernel.org/cgit/git/git.git/commit/?id=30ffa603 I found it mentioned in the release notes for git v1.5.5. https://git.kernel.org/cgit/git/git.git/tree/Documentation/RelNotes/1.5.5.txt

What is the current minimum bar for `git` support?